[2025-03-24T13:05:23.160+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-24T13:05:23.184+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_first_dag_2.first_task manual__2025-03-24T13:05:19.612723+00:00 [queued]>
[2025-03-24T13:05:23.194+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_first_dag_2.first_task manual__2025-03-24T13:05:19.612723+00:00 [queued]>
[2025-03-24T13:05:23.195+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-03-24T13:05:23.219+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): first_task> on 2025-03-24 13:05:19.612723+00:00
[2025-03-24T13:05:23.231+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=108) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-03-24T13:05:23.233+0000] {standard_task_runner.py:72} INFO - Started process 109 to run task
[2025-03-24T13:05:23.233+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_2', 'first_task', 'manual__2025-03-24T13:05:19.612723+00:00', '--job-id', '31', '--raw', '--subdir', 'DAGS_FOLDER/fristdags.py', '--cfg-path', '/tmp/tmpqgfo4_tb']
[2025-03-24T13:05:23.234+0000] {standard_task_runner.py:105} INFO - Job 31: Subtask first_task
[2025-03-24T13:05:23.392+0000] {task_command.py:467} INFO - Running <TaskInstance: our_first_dag_2.first_task manual__2025-03-24T13:05:19.612723+00:00 [running]> on host e1537a8d388b
[2025-03-24T13:05:23.519+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='our_first_dag_2' AIRFLOW_CTX_TASK_ID='first_task' AIRFLOW_CTX_EXECUTION_DATE='2025-03-24T13:05:19.612723+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-24T13:05:19.612723+00:00'
[2025-03-24T13:05:23.520+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-24T13:05:23.591+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-03-24T13:05:23.592+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', "echo 'hello this first task'"]
[2025-03-24T13:05:23.606+0000] {subprocess.py:99} INFO - Output:
[2025-03-24T13:05:23.615+0000] {subprocess.py:106} INFO - hello this first task
[2025-03-24T13:05:23.615+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-03-24T13:05:23.655+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-24T13:05:23.656+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=our_first_dag_2, task_id=first_task, run_id=manual__2025-03-24T13:05:19.612723+00:00, execution_date=20250324T130519, start_date=20250324T130523, end_date=20250324T130523
[2025-03-24T13:05:23.735+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-24T13:05:23.753+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-03-24T13:05:23.819+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
